lib/cocoapods/project.rb

```diff
@@ -69,7 +69,7 @@ def initialize(path, skip_initialization = false,
     #
     def generate_available_uuid_list(count = 100)
       start = @generated_uuids.size
-      uniques = Array.new(count) { |i| format('%.4s%07X0', @uuid_prefix, start + i) }
+      uniques = Array.new(count) { |i| format('%.6s%07X0', @uuid_prefix, start + i) }
```

Comment:
I think i also need to bump 7 down to 5?

Reply:
/cc @segiddins

Reply:
you don't _have_ to